frontend/src/home/Home.tsx

```diff
@@ -215,9 +217,12 @@ class HomeComponent extends React.Component<Props> {
       // switched plan
       this.callUpdatePlanLastViewedOnInterval();
     }
+    window.addEventListener("beforeunload", this.sendPlanUpdates);
   }
 
   componentWillUnmount() {
+    this.sendPlanUpdates();
+    window.removeEventListener("beforeunload", this.sendPlanUpdates);
```

Comment:
We may want to keep an eye on this, I've heard people say that making an API call on `beforeunload` or `unload` is unreliable, but I think this is fine until we determine it's not.